HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -155,36 +160,74 @@ void HLTScoutingPFProducer::produce(edm::StreamID sid, edm::Event &iEvent, edm::
             break;
           ++index_counter;
         }
+        float normchi2{0}, dz{0}, dxy{0}, dzError{0}, dxyError{0}, trk_pt{0}, trk_eta{0}, trk_phi{0};
+        uint8_t lostInnerHits{0}, quality{0};
+        if (doTrackRelVars_) {
+          const auto *trk = cand.bestTrack();
+          if (trk != nullptr) {
+            normchi2 = MiniFloatConverter::reduceMantissaToNbitsRounding(std::floor(trk->normalizedChi2()),
```

Comment:
Last question from my side: why do you use `floor` here?

Reply:
It originally comes from the ParticleNet implementation (see [here](https://github.com/cms-sw/cmssw/blob/master/RecoBTag/FeatureTools/plugins/DeepBoostedJetTagInfoProducer.cc#L440)). However I will check why `floor` is used there.

Reply:
It seems like it was used to make things consistent between MiniAOD and AOD (as normalizedChi2 in MiniAOD was compressed and stored as an int, while in AOD it was a float). However I don't think that is relevant here, so I can remove it.

Reply:
I don't have a preference (mine was more of a curiosity). I think this choice is left to the judgment of you and other scouting experts. If you leave the `floor`, it would be useful to add a comment on the reason in the line above.

Reply:
I just removed it as to my understanding it isn't needed.